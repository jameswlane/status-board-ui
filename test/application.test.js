const index = require('../index');

describe('Test', () => {
  test('Test', () => {
    expect(index).toMatchSnapshot();
  });
});
