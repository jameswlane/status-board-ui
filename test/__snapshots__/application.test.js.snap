// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Test Test 1`] = `
Object {
  "application": "",
  "default": [Circular],
  "log": "",
  "vendor": "",
}
`;
